web_src/css/repo.css

```diff
@@ -3402,11 +3402,11 @@ tbody.commit-list {
   background: var(--color-card);
 }
 
-.pinned-issue-card .meta a {
+.pinned-issue-card a {
   color: inherit;
 }
 
-.pinned-issue-card .meta a:hover {
+.pinned-issue-card a:hover {
```

Comment:
I guess these styles for `a` could be done by the `muted` class on these `<a>` elements, then no need to write these styles?